uftrace.c

```diff
@@ -1484,6 +1490,9 @@ int main(int argc, char *argv[])
 	if (opts.mode == UFTRACE_MODE_TUI)
 		opts.srcline = true;
 
+	if (!opts.pid) /* Keep uninitialized values in client mode */
```

Comment:
Do we really need to keep the value?  It feels like the sensible default value is ON and we can change the state for uncommon case like client mode or explicit trace-off.

Reply:
When in client mode, we only want to forward the trace option to the agent went the user specifies `--trace=on|off`. Otherwise, when tracing is off, the default `trace=on` applies, so the user needs to use `--trace=off` explicitly so the tracing state doesn't change by itself.
The `TRACE_STATE_NONE` value means the user hasn't specified a new tracing state. I need to have it as a default value to know whether the `TRACE_STATE_ON` value comes from user action or is a default value. Do you see another solution?

Reply:
No, I think we're good.